server/controllers/routes/user/cohort/getSpecificCohort.js

```diff
@@ -0,0 +1,21 @@
+const {
+  cohort: { getSpecificCohort },
```

Comment:
I suggest making destructure of getSpecificCohort in **``` /database/queries/index ```** as :
```js
const { getSpecificCohort } = require('./cohort');

module.exports = { getSpecificCohort };
```
and here in this file you can make require to getSpecificCohort as 
```suggestion
  const { getSpecificCohort } = require('../../../../database/queries');
```
so we get more benefit from **``` /database/queries/index ```** file.